src/models/client.js

```diff
@@ -0,0 +1,40 @@
+import uuid from 'node-uuid';
+import Debug from 'debug';
+
+let clientStore = [];
```

Comment:
Make this an object or a map, shouldn't be a problem hashing them with a uuid, O(1) < O(n).